docs-markdown/src/helper/utility.ts

```diff
@@ -114,37 +113,24 @@ export function search(editor: vscode.TextEditor, selection: vscode.Selection, s
         for (const file in files) {
             if (files.hasOwnProperty(file)) {
                 const baseName: string = (path.parse(files[file]).base);
-                let fileName: string = files[file];
-                fileName = fileName.toLowerCase();
+                const fileName: string = files[file];
                 if (fileName.includes(searchTerm)) {
```

Comment:
Looks like the `toLowerCase` was removed for the search term match. On Windows, this should yield the same results, but that might not be the case on Linux or Mac (where directories and filenames are case-sensitive). Just checking to see whether this was part of the criteria.

Reply:
We talked about this in person, so I'm approving.